pkg/log/log.go

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import (
+	"fmt"
+
+	"github.com/go-logr/logr"
+)
+
+type ResourceChangeType string
+
+const (
+	ResourceCreate   ResourceChangeType = "Create"
+	ResourceUpdate   ResourceChangeType = "Update"
+	ResourceDelete   ResourceChangeType = "Delete"
+	ResourceComplete ResourceChangeType = "Complete"
```

Comment:
Can we get a comment describing what "Complete" means?  I think it's only used in the context of CDQ but it would be good to be clear about this

Reply:
updated